src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTOTest.kt

```diff
@@ -94,6 +94,29 @@ class DraftReferralDTOTest(@Autowired private val json: JacksonTester<DraftRefer
     assertThat(draftReferral.relevantSentenceId).isEqualTo(123456789)
   }
 
+  @Nested inner class RiskInformation {
+    // this field has some funky JsonView annotations going on,
+    // so double check it's working as expected
+
+    @Test
+    fun `test serialization of additionalRiskInformation in draft referrals`() {
+      val referral = referralFactory.createDraft()
+
+      val out = json.write(DraftReferralDTO.from(referral))
+      assertThat(out).hasJsonPath("$.additionalRiskInformation")
```

Comment:
Is it worth testing non-null values too (to assert it serialises as expected)?

Reply:
sure thing - done